src/PlatformJson.js

```diff
@@ -27,11 +27,11 @@ class PlatformJson {
     }
 
     static load (plugins_dir, platform) {
-        const filePath = path.join(plugins_dir, `${platform}.json`);
-        let root = null;
-        if (fs.existsSync(filePath)) {
-            root = JSON.parse(fs.readFileSync(filePath, 'utf-8'));
-        }
+        const filePath = path.join(plugins_dir, platform + '.json');
```

Comment:
```suggestion
        const filePath = path.join(plugins_dir, `${platform}.json`);
```

Seemed like an accidental revert of the template literal.